### PR TITLE
Changed to version 2.0.2. Implemented 'readTimeout' and 'connectionTi…

### PR DESCRIPTION
**Pull request by austospumanto**
…meout' options for the cloudhub connector to deal with server-side timeouts. Made changes to mulesoft/ion-client-api project to deal with these new options.

https://www.mulesoft.org/jira/browse/CHUBC-9

### CODE REVIEW

**Comment by jeff-1amstudios** [comment]
@estebanwasinger could you review this - you know this much better than me :)

**Comment by estebanwasinger** [comment]
sure 👍

File: src/main/java/org/mule/modules/cloudhub/config/BasicAuthConfig.java [hunk 1]
```diff
@@ -55,13 +55,16 @@ public void setMaxWaitTime(Long maxWaitTime) {
...
      * @param sandbox SandBox name
+     * @param connectionTimeout Specifies the amount of time, in milliseconds, that the consumer will attempt to establish a connection before it times out. Default value is 0, which means infinite
+     * @param readTimeout Specifies the amount of time, in milliseconds, that the consumer will wait for a response before it times out. Default value is 0, which means infinite
      * @throws ConnectionException
      */
     @Connect
     @TestConnectivity
-    public void connect(@ConnectionKey String username, @Password String password, @Default("https://anypoint.mulesoft.com/cloudhub/") String url, @Optional String sandbox) throws ConnectionException {
+    public void connect(@ConnectionKey String username, @Password String password, @Default("https://anypoint.mulesoft.com/cloudhub/") String url, @Optional String sandbox,
+    					@Default("0") Integer connectionTimeout, @Default("0") Integer readTimeout) throws ConnectionException {
```
Comment (estebanwasinger) [inline src/main/java/org/mule/modules/cloudhub/config/BasicAuthConfig.java]: `connectionTimeout ` and `readTimeout` should be `@Configurable` fields in the same way as `TokenConfig` does. This fields are injected before the `connect()` method is called

File: pom.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mule.modules</groupId>
     <artifactId>mule-module-cloudhub</artifactId>
-    <version>2.0.1</version>
+    <version>2.0.2</version>
```
Comment (estebanwasinger) [inline pom.xml]: This should be merged into develop branch with a SNAPSHOT version and check that CI runs all the test correctly.

Comment (estebanwasinger) [inline pom.xml]: Change DevKit version from `3.7.2` to `3.9.0`

File: src/main/java/org/mule/modules/cloudhub/config/BasicAuthConfig.java (hunk 1 above)
Comment (austospumanto) [inline src/main/java/org/mule/modules/cloudhub/config/BasicAuthConfig.java]: Changed.

**Pull request closed without merging**